client/src/pages/Contest/ContestForm/ContestForm.tsx

```diff
@@ -0,0 +1,273 @@
+import { useState } from 'react';
+import { Formik } from 'formik';
+import momentTimezone from 'moment-timezone';
+import moment, { Moment } from 'moment';
+import MomentUtils from '@date-io/moment';
+import * as Yup from 'yup';
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import { tatooDesigns as images } from './testdata.js';
+import checkmark from './checkmark.png';
+import createDeadlineDate from './helpers/createDeadlineDate.js';
+import {
+  CircularProgress,
+  Grid,
+  InputAdornment,
+  Box,
+  TextField,
+  Button,
+  Paper,
+  Select,
+  MenuItem,
+  FormControl,
+  ImageList,
+  ImageListItem,
+} from '@material-ui/core';
+import { KeyboardDatePicker, KeyboardTimePicker } from '@material-ui/pickers';
+
+interface MyFormValues {
+  title: string;
+  description: string;
+  prizeAmount: number;
+  date: MaterialUiPickersDate | Moment | string | null;
+  time: MaterialUiPickersDate | Moment | string | null;
+  timeZone: string;
+  imageFiles: string[];
+}
+
+export const ContestForm: React.FC = () => {
+  const classes = useStyles();
+  const [selectedImages, setSelectedImages] = useState<string[]>([]);
+  const [selectedDate, handleDateChange] = useState<MaterialUiPickersDate | Moment>(moment());
```

Comment:
Can we can use the same state for Date/Time (A JS object contain both date/time, I guess even a moment object is the same)?

Reply:
I will definitely try to manage Date/Time with one state. It will be much more simpler and cleaner. I'll implement the request changes and merge it by tonight. Thank you.